src/app/footer/footer.component.html

```diff
@@ -2,7 +2,7 @@
     <a target="_blank" rel="noreferrer" href="https://mon.gov.ua/">
       <img src="../../assets/images/MES.png" alt="Ministry of Education">
     </a>
-    <a target="_blank" rel="noreferrer"  href="https://www.softserveinc.com/">
-      <img src="../../assets/images/SS.png"   alt="SoftServe">
+    <a target="_blank" rel="noreferrer" href="https://www.softserveinc.com/">
+      <img src="../../assets/images/SS.png" alt="SoftServe">
```

Comment:
try to avoid ss namings due to company policies

Reply:
I didn't write this code (only deleted extra spaces)

Reply:
follow trail of goodness =)